docs/for-devs/console/usermgnt.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: user-management
+title: Team - User Management
+sidebar_label: User Management
+---
+
+## About User Management
+
+What you need to know about the User Management feature:
+
+- Only platform administrators can create or delete users. Ask your Platform Administrator to create an account for a new Team member.
+- Platform admin can assign users the Team admin role.
+- Team admins can assign existing users to get access to the Team they administer.
+- Team members are not able to see the User Management section in the menu.
+- A password reset can only be performed by a Platform Administrator.
```

Comment:
At the moment Platform Administrator is not able to login to keycloak admin console. As an alternative platform admin can re-create a user. (remove, deploy changes, add, deploy changes)